### PR TITLE
[IAM-5169] add docs for the migration process to the new SSO

### PR DESCRIPTION
**Pull request by przemek-sl**
I'm adding documentation on the migration process from the deprecated SSO to the new global SSO.

### CODE REVIEW

File: docs/basics/sso/setting-up-sso.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <p><span className="sauceGreen">Enterprise Plans only</span></p>
 
+:::caution
+If you have previously implemented the [deprecated SSO integration](/basics/sso-deprecated/setting-up-single-sign-on/) and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
Comment (aliciakrebs) [inline docs/basics/sso/setting-up-sso.md:12]:
```suggestion
If you have previously implemented the [deprecated SSO integration](/basics/sso-deprecated/setting-up-single-sign-on/) and wish to migrate to the new SSO implementation,  follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/).
```

File: docs/basics/acct-team-mgmt/org-settings.md [hunk 2]
```diff
@@ -57,7 +57,9 @@ To access Security settings:
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
```
Comment (aliciakrebs) [inline docs/basics/acct-team-mgmt/org-settings.md:60]:
```suggestion
If you do not have any SSO integrations set up at Sauce Labs and you wish to establish a new integration, see [Setting Up SSO](/basics/sso/setting-up-sso) to get started.
```

File: docs/basics/acct-team-mgmt/org-settings.md [hunk 3]
```diff
@@ -57,7 +57,9 @@ To access Security settings:
...
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
Comment (aliciakrebs) [inline docs/basics/acct-team-mgmt/org-settings.md:62]:
```suggestion
If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, see the [step-by-step migration guide](/basics/sso/migration-from-deprecated-sso/).
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+id: migration-from-deprecated-sso
+title: Migration from the deprecated SSO
+sidebar_label: Migration from the Deprecated SSO
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+The following document will help you migrate smoothly from the [deprecated SSO](/basics/sso-deprecated/setting-up-single-sign-on/) to the [new SSO](/basics/sso/setting-up-sso/).
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:11]:
```suggestion
This guide explains how to migrate from the [deprecated SSO](/basics/sso-deprecated/setting-up-single-sign-on/) to the [new SSO](/basics/sso/setting-up-sso/).
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+| JIT Provisioning                  | In the new SSO, JIT (Just-In-Time) provisioning is enabled by default and cannot be disabled. This means that if a user from your Identity Provider (IdP) does not have a Sauce Labs account, one will be automatically created for them. <br/><br/> However, in the legacy SSO, some organizations requested to disable this feature. In such cases, it's important to note that users from your IdP who do not have accounts in Sauce Labs will still be able to provision accounts automatically. Please keep this in mind during the migration process from legacy SSO to the new SSO. |
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:28]:
```suggestion
| JIT Provisioning                  | In the new SSO, JIT (Just-In-Time) provisioning is enabled by default and cannot be disabled. This means that if a user from your Identity Provider (IdP) does not have a Sauce Labs account, one will be automatically created for them. <br/><br/> However, in the depreciated  SSO, it was possible to disable this feature. In such cases, it's important to note that users from the IdP who do not have accounts in Sauce Labs will still be able to provision accounts automatically. Keep this in mind during the migration process from deprecated  SSO to the new SSO. |
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+| Enforce SAML SSO                  | This [setting](/basics/sso/setting-up-sso/#enforce-saml-sso) is also present in the new SSO, but it is common to both the new SSO and the legacy SSO. When turned on, it allows authentication only via either the new SSO or the legacy SSO. When turned off, users can also log in via username and password, in addition to SSO authentication.                                                                                                                                                                                                                                         |
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:29]: suggest to change legacy to depreciated in all places for consistency (or depreciated to legacy if you prefer) 
```suggestion
| Enforce SAML SSO                  | This [setting](/basics/sso/setting-up-sso/#enforce-saml-sso) is also present in the new SSO, but it is common to both the new SSO and the legacy SSO. When turned on, it allows authentication only via either the new SSO or the legacy SSO. When turned off, users can also log in via username and password, in addition to SSO authentication.                                                                                                                                                                                                                                         |
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+2. On the **Organization Management** page, in the **ORGANIZATION SETTINGS** section, click **View Settings**.
+
+<img src={useBaseUrl('img/basics/sso/setup-sso-sp-org-settings-btn.png')} alt="Organization Setting Button" width="800" />
+
+3. On the **Organization Settings** page, click the **SINGLE SIGN-ON** tab.
+
+<img src={useBaseUrl('img/basics/sso/setup-sso-sp-sso-tab.png')} alt="SSO Tab" width="400" />
+
+4. If your organization is eligible for migrating to the new SSO, you will see the following banner. Please note that the migration wizard may not be enabled for your organization yet. If you don't see the banner, it means our development team is still working on enabling the migration wizard for your organization.
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:52]: how long should the user wait to let us know that the banner is not available? Is there a timeframe or a contact for this? 
```suggestion
4. If your organization is eligible for migrating to the new SSO, you will see the following banner. Please note that the migration wizard may not be enabled for your organization yet. If you don't see the banner, it means our development team is still working on enabling the migration wizard for your organization.
```

File: docs/basics/sso/setting-up-sso.md (hunk 1 above)
Comment (przemek-sl) [reply docs/basics/sso/setting-up-sso.md:12]: fixed

File: docs/basics/acct-team-mgmt/org-settings.md (hunk 2 above)
Comment (przemek-sl) [reply docs/basics/acct-team-mgmt/org-settings.md:60]: fixed

File: docs/basics/acct-team-mgmt/org-settings.md (hunk 3 above)
Comment (przemek-sl) [reply docs/basics/acct-team-mgmt/org-settings.md:62]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 4 above)
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:11]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 5 above)
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:28]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 6 above)
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:29]: ok, for consistency I will not use "legacy" but I thought that the word "deprecated" is more appropriate in this context than "depreciated". What do you suggest?

source: https://www.baeldung.com/cs/deprecated-vs-obsolete

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 7 above)
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:52]: The first group (almost 300 organizations) which will go through "the seamless migration" will see the banner soon, that is I hope in two weeks. The automatic migration for other organizations (less than 100) is not yet prepared. There are some extra actions that need to be taken before these organization can be qualified for "the seamless migration". The timeframe is not yet defined.

[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:52]: fixed

[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:52]: fixed

File: docs/basics/sso/setting-up-sso.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/setting-up-sso.md:12]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+sidebar_label: Migration from the Deprecated SSO
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+The following document will help you migrate smoothly from the [deprecated SSO](/basics/sso-deprecated/setting-up-single-sign-on/) to the [new SSO](/basics/sso/setting-up-sso/).
+
+Before you begin the migration process, please review the differences between the two implementations, which are listed below.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:13]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:28]: fixed

File: docs/basics/sso-deprecated/config-adfs.md [hunk 9]
```diff
@@ -9,7 +9,9 @@ sidebar_label: Configuring Active Directory Federation Services (AD FS)
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/config-adfs.md:14]: fixed

File: docs/basics/data-center-endpoints.md [hunk 10]
```diff
@@ -171,7 +171,9 @@ If you use any of the following and your data center is EU Central, you need to
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/data-center-endpoints.md:176]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+:::tip
+Both the deprecated SSO integration and the new SSO integration can work independently and concurrently. This means that you can set up the new SSO and test it while your users continue to use the deprecated integration.
+
+You will have access to the configuration panels of both integrations in the Team Management UI, allowing you to manage and configure them separately.
+:::
+
+## Differences in the new SSO
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md:21]: fixed

File: docs/basics/sso-deprecated/sso-hub.md [hunk 12]
```diff
@@ -9,7 +9,9 @@ sidebar_label: Getting Started
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/sso-hub.md:14]: fixed

File: docs/basics/sso-deprecated/setting-up-single-sign-on.md [hunk 13]
```diff
@@ -4,16 +4,38 @@ title: Setting Up Single Sign-On (Deprecated Flow)
...
 <p><span className="sauceGold">Deprecated</span></p>
+<p><span className="sauceGreen">Enterprise Plans only</span></p>
 
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/setting-up-single-sign-on.md:17]: fixed

File: docs/basics/sso-deprecated/setting-up-single-sign-on.md [hunk 14]
```diff
@@ -4,16 +4,38 @@ title: Setting Up Single Sign-On (Deprecated Flow)
...
+:::
+<br/>
+
+:::tip
+
+### How do I know if my organization uses the deprecated SSO integration?
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/setting-up-single-sign-on.md:23]: fixed

File: docs/basics/sso-deprecated/setting-up-single-sign-on.md [hunk 15]
```diff
@@ -4,16 +4,38 @@ title: Setting Up Single Sign-On (Deprecated Flow)
...
+
+- The Service Provider entity ID (Issuer) in your Identity Provider configuration is set to either `https://saucelabs.com/sso/metadata` or `https://eu-central-1.saucelabs.com/sso/metadata`.
+- The Assertion Consumer Service (ACS) URL in your Identity Provider configuration is set to one of the following values:
+  - `https://app.saucelabs.com/sso/acs`
+  - `https://saucelabs.com/sso/acs`
+  - `https://app.eu-central-1.saucelabs.com/sso/acs`
+  - `https://eu-central-1.saucelabs.com/sso/acs`
+- The configuration panel for the deprecated SSO is different from the new one. [The legacy one](https://docs.saucelabs.com/basics/acct-team-mgmt/org-settings/#single-sign-on-settings-deprecated-flow) looks as following:
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/setting-up-single-sign-on.md:31]: fixed

File: docs/basics/sso-deprecated/config-okta.md [hunk 16]
```diff
@@ -9,7 +9,9 @@ sidebar_label: Configuring Okta
 :::caution
 This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.
 
-If you want to set up a new SSO integration, please use [the new SSO implementation](/basics/sso/setting-up-sso).
+If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.
+
+If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso-deprecated/config-okta.md:14]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+
+Before you begin the migration process, review the differences between the two implementations listed below.
+
+:::tip
+Both the deprecated SSO integration and the new SSO integration can work independently and concurrently. This means that you can set up the new SSO and test it while your users continue to use the deprecated integration.
+
+You will have access to the configuration panels of both integrations in the Team Management UI, allowing you to manage and configure them separately.
+:::
+
+## Differences In The New SSO
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:21]:
```suggestion
## Differences In the New SSO
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+| JIT Provisioning                  | In the new SSO, JIT (Just-In-Time) provisioning is enabled by default and cannot be disabled. This means that if a user from your Identity Provider (IdP) does not have a Sauce Labs account, one will be automatically created for them. <br/><br/> However, in the deprecated  SSO, it was possible to disable this feature. In such cases, it's important to note that users from the IdP who do not have accounts in Sauce Labs will still be able to provision accounts automatically. Keep this in mind during the migration process from deprecated  SSO to the new SSO. |
```
Comment (aliciakrebs) [inline docs/basics/sso/migration-from-deprecated-sso.md:28]:
```suggestion
| JIT Provisioning                  | In the new SSO, JIT (Just-In-Time) provisioning is enabled by default and cannot be disabled. This means that if a user from your Identity Provider (IdP) does not have a Sauce Labs account, one will be automatically created for them. <br/><br/> However, in the deprecated  SSO, it was possible to disable this feature. In such cases, it's important to note that users from the IdP who do not have accounts in Sauce Labs will still be able to provision accounts automatically. Keep this in mind during the migration process from deprecated SO to the new SSO. |
```

File: docs/basics/sso/migration-from-deprecated-sso.md [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md]: fixed

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 6 above)
Comment (aliciakrebs) [reply docs/basics/sso/migration-from-deprecated-sso.md:29]: yes, that was an autocorrect typo - this should be deprecated.

**Review by aliciakrebs — APPROVED** [review]

File: docs/basics/sso/migration-from-deprecated-sso.md (hunk 6 above)
Comment (przemek-sl) [reply docs/basics/sso/migration-from-deprecated-sso.md]: fixed

**Pull request merged**